src/scripts/list-users.ts

```diff
@@ -15,6 +15,10 @@ import {
   GroupService,
   Group,
 } from './services/group-service';
+import {
+  validateOktaServerIsRunning,
```

Comment:
**Issue:** don't import from 'peers' like this. This isn't necessary for now, I would rather rewrite the ticket than introduce this change.

Reply:
modified in commit: 86b86bb42c3c933247cfd80ab13535eb1eed5de5